hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/SearchDataClassOrderer.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search
+
+import org.junit.jupiter.api.ClassDescriptor
+import org.junit.jupiter.api.ClassOrderer
+import org.junit.jupiter.api.ClassOrdererContext
+
+class SearchDataClassOrderer : ClassOrderer {
+  override fun orderClasses(classOrdererContext: ClassOrdererContext) {
+    classOrdererContext.classDescriptors.sortWith(Comparator.comparingInt { getOrder(it) })
+  }
+
+  private fun getOrder(classDescriptor: ClassDescriptor): Int =
+    if (AbstractSearchDataIntegrationTest::class.java.isAssignableFrom(classDescriptor.testClass)) 1 else 2
```

Comment:
ensure that the data tests are all run after each other so that we don't get issues with test pollution